components/network/rsync/Makefile

```diff
@@ -1,22 +1,12 @@
 #
-# CDDL HEADER START
+# This file and its contents are supplied under the terms of the
```

Comment:
Is it okay to change the copyright notice of already existing file with the Oracle copyright in it?

Reply:
The original text references http://www.opensolaris.org/os/licensing which is not available anymore. The new text is the best replacement as it seems to be in the same vein and continues to use the CDDL.